laletterbuilder/letter_sending.py

```diff
@@ -130,11 +132,10 @@ def send_letter(letter: models.Letter):
     letter_type = letter.get_letter_type()  # TODO: localize this somewhere
     ld = user.landlord_details
 
-    # TODO: fill in user pref
     if ld.email:
```

Comment:
And here's where we'll have to check not just if we have a LL email, but if the user wants us to send it. (per our convo with Leslie)